test/unit/language-test.js

```diff
@@ -337,6 +341,30 @@ describe('languageUtils', function() {
             expect(customLocalization.pause).to.equal(localizationPause);
             expect(customLocalization.stop).to.equal(localizationStop);
         });
+
+        it('should only use custom localization block if "forceLocalizationDefaults" is true', function() {
+            stubHtmlLanguage(document, 'fr');
+            const config = new Config({
+                forceLocalizationDefaults: true,
+                localization,
+                intl
+            });
+            const configLocalization = config.localization;
+            expect(getLanguage()).to.equal('fr');
+            expect(configLocalization.play).to.equal(localizationPlay);
+            expect(configLocalization.pause).to.equal(localizationPause);
+            expect(configLocalization.stop).to.equal(localizationStop);
+            
+            Object.keys(en).forEach(key => {
+                if (key === 'play' || key === 'pause' || key === 'stop') {
```

Comment:
we don't need this if block right ? Since what we care is that all localization values match the default values.
I assume you had this because those values are the same in our french translation, but that doesn't matter.